### DISCUSSION TITLE
Autobind not working

### DISCUSSION THREAD

**Discussion by itsNotMyUsername** [99designs/gqlgen#2230] [Q&A]
### What happened?
I copied `Message` type from https://github.com/99designs/gqlgen/blob/master/codegen/config/testdata/autobinding/chat/message.go to my public temporary repository https://github.com/itsNotMyUsername/example/blob/main/chat/message.go
Passing the url to autobind:

```
autobind:
  - github.com/itsNotMyUsername/example/chat
 ```
results in
`generating core failed: unable to load github.com/itsNotMyUsername/example/chat - make sure you're using an import path to a package that exists`.

If I pass url to autobind to original model like so:

```
autobind:
  - github.com/99designs/gqlgen/codegen/config/testdata/autobinding/chat
```
all works as expected, types are reused and imported in generated resolver from above url.

What am I missing that autobinds is not working for my repo with same model?

### What did you expect?

The model to be reused and no error thrown.

### Minimal graphql.schema and models to reproduce
```
extend type Query {
	fetch(input: Message): Boolean!
}

input Message {
    name: String
}
```

### versions
 - `go run github.com/99designs/gqlgen version`? v0.17.9
 - `go version`? go1.18.1

**Comment by frederikhors** [comment]
Please, do not open the same issue again. Autobind works. Read docs carefully or wait if someone can help you.

**Reply by Micrococonut** [reply]
Can you post the relevant documentation for this feature? Control+F on every page of gqlgen.com brought up exactly 1 result. The one sentence explanation in gqlgen.yml. So can you please post the relevant documentation you are referring to that is supposed to explain how this feature actually works?

**Comment by itsNotMyUsername** [comment]
Apologies, it didnt look like the same issue. Initially I thought its only related to private repos but looks like it doesnt work for me with public repo either.

What am I missing from docs https://gqlgen.com/config/ under autobind section that I missed to do?
```
# gqlgen will search for any type names in the schema in these go packages
# if they match it will use them, otherwise it will generate them.
```

**Reply by ibadi-id** [reply]
you are missing double quote ""
```
autobind:
  - "github.com/itsNotMyUsername/example/chat"
 ```
